Src/Sarif.PatternMatcher.Function/Sarif.PatternMatcher.Function.csproj

```diff
@@ -25,19 +25,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
-    <None Update="Rules\*.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
-  
-  <ItemGroup>
-    <SpamFiles Include="$(MSBuildThisFileDirectory)..\Plugins\Security\*.json" />
   </ItemGroup>
   
   <ItemGroup>
+    <ProjectReference Include="..\Plugins\Security\Security.csproj" />
     <ProjectReference Include="..\Sarif.PatternMatcher\Sarif.PatternMatcher.csproj" />
   </ItemGroup>
-  <Target Name="CopyingSpamRules" BeforeTargets="Build">
-    <Copy SourceFiles="@(SpamFiles)" DestinationFolder="$(MSBuildThisFileDirectory)\Rules\" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false" />
-  </Target>
```

Comment:
We won't need this, since I added a project reference to always publish the security.csproj as well